### PR TITLE
Add landing page for Porto 2023 hackathon

### PR DESCRIPTION
**Pull request by razvand**
Hackathon takes place at Faculdade de Engenharia Universidade do Porto in May 2023.

### CODE REVIEW

File: content/en/community/hackathons/2023-05-porto/_index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+| ------------- | -------------------------------------------------------------------- |
+| 10:00 - 10:15 | Overview of Unikernels and Unikraft                                  |
+| 10:15 - 11:15 | Introduction to Unikraft: Building and Running Standard Applications |
+| 11:15 - 11:30 | Break                                                                |
+| 11:30 - 12:45 | Unikraft Internals: Building, Configuring, Using Libraries           |
+| 12:45 - 14:00 | Lunch                                                                |
+| 14:00 - 15:00 | Running (Complex) Applications in Unikraft                           |
+| 15:00 - 16:00 | Porting Applications to Unikraft                                     |
+| 16:00 - 16:15 | Break                                                                |
+| 16:15 - 17:45 | Porting Applications to Unikraft                                     |
```
Comment (StefanJum) [inline content/en/community/hackathons/2023-05-porto/_index.md:52]: There are two of  `Porting Applications to Unikraft `. Should one of them be `Binary Compatibility`?

File: content/en/community/hackathons/2023-05-porto/_index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+### People
+
+The hosts of the hackathon are:
+
+* [João Cardoso](https://sigarra.up.pt/feup/en/func_geral.formview?p_codigo=449856)
+* [Pedro Diniz](https://sigarra.up.pt/feup/pt/func_geral.formview?p_codigo=672712)
+* [Pedro Souto](https://sigarra.up.pt/feup/en/func_geral.formview?p_codigo=238172)
+* [João Mendes Moreira](https://dei.fe.up.pt/mdse/portfolio/director/)
+* [Tiago Filipe Castro Viana](neacm@fe.up.pt)
```
Comment (StefanJum) [inline content/en/community/hackathons/2023-05-porto/_index.md:34]: The email will redirect to `http://localhost:1313/neacm@fe.up.pt`, which does not work.

**Review by StefanJum — COMMENTED** [review]
@razvand see the few comments. All good besides that.

File: content/en/community/hackathons/2023-05-porto/_index.md (hunk 2 above)
Comment (razvand) [reply content/en/community/hackathons/2023-05-porto/_index.md:34]: I see, will fix.

File: content/en/community/hackathons/2023-05-porto/_index.md (hunk 1 above)
Comment (razvand) [reply content/en/community/hackathons/2023-05-porto/_index.md:52]: They're both going to be on that. But the term `Porting Applications to Unikraft` is more encompassing.

Comment (razvand) [reply content/en/community/hackathons/2023-05-porto/_index.md:52]: Or anything related to application porting, actually. That's why I would leave it as such, to fit most approaches.

File: content/en/community/hackathons/2023-05-porto/_index.md (hunk 2 above)
Comment (razvand) [reply content/en/community/hackathons/2023-05-porto/_index.md]: fixed

**Review by StefanJum — APPROVED** [review]
All good now, thanks.

Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>
Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Pull request merged**